package.json

```diff
@@ -3,14 +3,15 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "node node_modules/react-native/local-cli/cli.js start",
-    "android": "node node_modules/react-native/local-cli/cli.js run-android",
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
     "test": "jest",
+    "lint": "eslint .",
```

Comment:
Does it work the same?

Reply:
It's going to work the same way for the most part except for that it's going to check the index.js file as well instead of ignoring it